osmosis-1/osmosis.zone.json

```diff
@@ -1658,6 +1658,16 @@
       "osmosis_main": false,
       "osmosis_frontier": true,
       "osmosis_info": true
+    },
+    {
+      "chain_name": "juno",
+      "base_denom": "cw20:juno1lrkn0zcc647pupzr5z67ru7qnyt2pze4szfxt5qqc003zquxkc3shxaks4",
+      "pools": {
+        "OSMO": 961
+      },
+      "osmosis_main": false,
+      "osmosis_frontier": true,
+      "osmosis_info": true
```

Comment:
the osmosis_info tag requires at least 1000 $liquidity in one off the pools listed above